src/sidebar/components/focused-mode-header.js

```diff
@@ -16,30 +18,47 @@ function FocusedModeHeader() {
     },
     selectors: {
       focusModeFocused: store.focusModeFocused,
+      focusModeHasUser: store.focusModeHasUser,
       focusModeUserPrettyName: store.focusModeUserPrettyName,
     },
   }));
 
+  // Nothing to do here for now if we're not focused on a user
```

Comment:
I've added this little guard here because we shouldn't render if we don't have a user to work with. This is a little pedantic as the only focused mode we support at present is on a user, but we don't want to assume...